<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookmarks</class>
 <widget class="QWidget" name="bookmarks">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>549</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>549</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>My Bookmarks</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel, QCheckBox, QTextEdit {
color: #fafafa;
}
</string>
  </property>
  <widget class="QListWidget" name="my_bookmarks_list">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
	color: rgb(0, 0, 0);
	border-radius: 5px;
	background-color: rgb(250, 250, 250);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="current_page_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Page:</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_bookmarks_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>My Bookmarks</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="current_page_textbox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:disabled {
	background-color: rgb(197, 197, 197);
}

QLineEdit {
	background-color: rgb(250, 250, 250);
	border-radius: 4px;
	border: 0;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookmark_name_textbox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:disabled {
	background-color: rgb(197, 197, 197);
}

QLineEdit {
	background-color: rgb(250, 250, 250);
	border-radius: 4px;
	border: 0;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookmark_name_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_bookmark_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Bookmark</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_bookmark_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>480</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visit_bookmark_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Visit Bookmark</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>551</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>481</width>
     <height>551</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
color: #fafafa;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_bookmark_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>my_bookmarks_list</zorder>
  <zorder>current_page_label</zorder>
  <zorder>my_bookmarks_label</zorder>
  <zorder>current_page_textbox</zorder>
  <zorder>bookmark_name_textbox</zorder>
  <zorder>bookmark_name_label</zorder>
  <zorder>add_bookmark_button</zorder>
  <zorder>delete_bookmark_button</zorder>
  <zorder>return_button</zorder>
  <zorder>visit_bookmark_button</zorder>
  <zorder>edit_bookmark_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
